src/App.js

```diff
@@ -1,29 +1,22 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+/* eslint-disable */
+import Calculator from './components/Calculator';
+/* eslint-enable */
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="main-container">
+        <Calculator />
+      </div>
+    );
+  }
 }
 
```

Comment:
## Your MIT :memo:
![MIT md](https://user-images.githubusercontent.com/93039359/184026034-d91dc0f3-612d-4c74-8fe1-7baf925b602d.png)

- [ ] Please add **your name**, **current year**, and **app type** in your MIT.md file. This is the license file of your project so please take it seriously :thinking: